Cli-CredentialHelper.Test/OperationArgumentsTests.cs

```diff
@@ -91,35 +50,32 @@ public void EmailAsUserName()
             var input = new InputArg
             {
                 Host = "example.visualstudio.com",
-                Password = "ḭncorrect",
+                Password = "incorrect",
```

Comment:
Why was this changed?

Reply:
Test EmailAsUserName seems to be copied from test SpecialCharacters, which tests password and username with special characters: i with tilde below in password, e with circumflex and hook above in username. Test EmailAsUserName should not repeat this specific test case for password, and stay limited to email as username.